scripts/install_core_dependencies.sh

```diff
@@ -17,4 +17,4 @@ sudo ${root_folder}/scripts/install_colmap.sh
 python3 -m pip install git+https://github.com/cvg/Hierarchical-Localization.git@v1.4
 
 # Pyceres.
-python3 -m pip install git+https://github.com/cvg/pyceres.git@v1.0
+python3 -m pip install git+https://github.com/cvg/pyceres.git@v2.3
```

Comment:
@pablospe did you try running the full pipeline on the test data?

I was installing latest version of pyceres in my first iterations and it was building but when I ran the pipeline `lamar.run` on the CAB data it was crashing in the middle.

Just want to warn you that this happened to me before.

Reply:
ok, let me revert this change.

Reply:
@mihaidusmanu do you know what part isn't working? Wondering if it would be easy to update the code to make work with last version.

Reply:
```suggestion
python3 -m pip install git+https://github.com/cvg/pyceres.git@v1.0
```

Reply:
@pablospe not sure, but I wouldn't change any of the original versions without 1. rerunning pseudo-GT pipeline on some data 2. running some evals on the lamar release. There's quite a few things that can go wrong.